pytest_container/container.py

```diff
@@ -777,102 +905,138 @@ def prepare_container(
             self.container_id, self.url = base.container_id, base.url
             return
 
-        with tempfile.TemporaryDirectory() as tmpdirname:
-            containerfile_path = join(tmpdirname, "Dockerfile")
-            iidfile = join(tmpdirname, str(uuid4()))
-            with open(containerfile_path, "w") as containerfile:
-                from_id = (
-                    self.base
-                    if isinstance(self.base, str)
-                    else (getattr(self.base, "url") or self.base._build_tag)
-                )
-                assert from_id
-                containerfile_contents = f"""FROM {from_id}
+        from_id = (
+            self.base
+            if isinstance(self.base, str)
+            else (getattr(self.base, "url") or self.base._build_tag)
+        )
```

Comment:
nit: are there any real benefits of using ternary if instead of a regular if/else statement?

I find this slightly unreadable (due to how it is formatted with `()`) and it does not achieve much unless it is very simple like:

`bla = foo if var else bar`